### PR TITLE
added protect explanation for master branches

### PR DESCRIPTION
**Pull request by regevran**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ A repository for working wg21 papers
 Each paper has its own master branch.
 Each paper resides in its own directory.
 
-### Branch Naming
+### Branches Naming
 Each paper is prefixed by `P<number>/`
 The main paper branch must be called `P<number>/master`.
 Working branches on a paper are freely named under `P<number>/<free-name>`
-Pull Requests for a paper are naturally to its master branch
+Pull Requests for a paper are naturally goes to its master branch.
```
Comment (danra) [inline README.md:12]: Was sort of correct grammatically before, now it isn't.
This works: "Pull Requests for a paper naturally go to its master branch."

File: README.md [hunk 2]
```diff
@@ -23,4 +23,5 @@ The working paper https://wg21.link/p0635 resides in `./P06535` and
 has a master branch `P06535/master`
 
 ## Managment
-Each paper owner is encaurged to add a restriction not allowing to commit directly to `P<number>/master`
+Branches following the `P<number>/master` format are protected from direct commits. Only Pull-Requests are allowed.
```
Comment (danra) [inline README.md:26]: Pull Requests, not Pull-Requests (to be consistent with above)

File: README.md [hunk 3]
```diff
@@ -23,4 +23,5 @@ The working paper https://wg21.link/p0635 resides in `./P06535` and
...
+`P<number>/master` is enforced by github via: `P[0-9][0-9][0-9][0-9]/master`
```
Comment (danra) [inline README.md:27]: This is fine, just commenting that we'll probably need an extra digit in a couple of years :)

File: README.md [hunk 4]
```diff
@@ -5,14 +5,14 @@ A repository for working wg21 papers
 Each paper has its own master branch.
 Each paper resides in its own directory.
 
-### Branch Naming
+### Branches Naming
```
Comment (danra) [inline README.md:8]: I think the original is better, actually

Comment (regevran) [reply README.md:8]: ok. Original chosen.

File: README.md (hunk 2 above)
Comment (regevran) [reply README.md:26]: fixed, also changed to lower case

**Pull request merged**